service/templates/task.html

```diff
@@ -315,6 +315,10 @@ <h1 class="one-liner" id="task-title">
                         </div>
                       </td>
                     </tr>
+                    <tr ng-show="task.task_end_details.oom_killer.detected">
+                      <td class="icon"><i class="fa fa-exclamation"></i></td>
+                      <td>OOM Killer detected (<b>[[ task.task_end_details.oom_killer.pids.join(", ") ]]</b>)</td>
```

Comment:
Could we write "PIDs:" before the list of PIDs?

Reply:
done